ush/rocoto/setup_expt.py

```diff
@@ -149,6 +161,48 @@ def edit_baseconfig():
     nens = args.nens
     cdump = args.cdump
     gfs_cyc = args.gfs_cyc
+    partition = args.partition
+
+    # Set machine defaults
+    if machine is 'WCOSS_DELL_P3':
```

Comment:
@KateFriedman-NOAA 
You should be using `if machine == 'WCOSS_DELL_P3'` and **not** `if machine is 'WCOSS_DELL_P3'`.
The **is** relational evaluates to `True` if the two variables point to the same object.
The **==** relational evaluates to `True` if the variables are equal.

Reply:
Gotcha, thanks!